piperider_cli/cli.py

```diff
@@ -106,9 +110,11 @@ def init(**kwargs):
         console.print(config)
 
 
-@cli.command(short_help='Test Configuration')
+@cli.command(short_help='Check project configuration.')
 @add_options(debug_option)
 def diagnose(**kwargs):
+    'Check project configuration, datasource, connections, assertion config and dbt manifest.'
```

Comment:
stick to `configuration` or `config` for consistency
for both project and assertion

Reply:
fixed, use configuration.